src/validation/validator-name.ts

```diff
@@ -36,12 +36,34 @@ export default class ValidatorName implements IValidator {
   }
 
   private titleCase(value: string): string {
-    const words = value.toLowerCase().split(' ');
-    const titleCase = (word: string) => word[0].toUpperCase() + word.slice(1);
-    const isRomanNumeral = /^[ivx]+$/ig;
-    const titleCased = words
+    if (!value) {
+      return '';
+    }
+
+    const titleCase = (word: string) => word[0]?.toUpperCase() + word.slice(1);
+    const isRomanNumeral = /^[ivx]+$/i;
```

Comment:
its clear and useful to have the regex captured via a descriptive name.  it would be nice to do the same for the regex on lines 56-58.  in general, i find regex hard to parse and read and benefit from either a description or a comment